components/dropdown/dropdown-content-mixin.js

```diff
@@ -575,11 +575,25 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 		if (newValue) {
 
-			if (ifrauBackdropService && this.mobileTray && this._useMobileStyling) {
-				this._ifrauContextInfo = await ifrauBackdropService.showBackdrop();
-			}
+			const evt = new CustomEvent('d2l-dropdown-before-open', {
```

Comment:
@margaree : I'm not sure where this event should be documented, given the way it is dispatched.  Does it need to be documented on the class headers (-content, -menu, -tabs), or can it be documented inline here?  I see a mix for the other events.

Reply:
Yup you're right - because of how it's dispatched, in a variable + separate `dispatchEvent`, it would need to be on the class headers for this mixin's consumers (e.g., https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-input.js#L15)